brave/src/main/java/com/linecorp/armeria/common/brave/RequestContextCurrentTraceContext.java

```diff
@@ -49,23 +49,24 @@
  */
 public final class RequestContextCurrentTraceContext extends CurrentTraceContext {
 
+    /**
+     * Sets whether the current thread is not a request thread, meaning it is never executed in the scope of a
+     * server or client request and will never have a {@link RequestContext} available. This can be called from
+     * background threads, such as the thread that reports traced spans to storage, to prevent logging a
+     * warning when trying to start a trace without having a {@link RequestContext}.
+     */
+    public static void setCurrentThreadNotRequestThread(boolean value) {
+        THREAD_NOT_REQUEST_THREAD.set(value);
```

Comment:
How about setting `TRUE` when `value` is `true` and removing when `false`?

Reply:
Ah yeah good point.